app/src/main/java/tech/relaycorp/gateway/ui/settings/LicensesActivity.kt

```diff
@@ -0,0 +1,13 @@
+package tech.relaycorp.gateway.ui.settings
+
+import android.os.Bundle
+import com.mikepenz.aboutlibraries.ui.LibsActivity
+import tech.relaycorp.gateway.R
+
+class LicensesActivity : LibsActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
Can we limit the dependencies to runtime ones, excluding test and built-time dependencies?

Reply:
Test dependencies only show up in `debug` builds, according to the documentation.

Reply:
Nice!